deck-rest/src/main/kotlin/com/deck/rest/builder/TeamBuilders.kt

```diff
@@ -0,0 +1,119 @@
+package com.deck.rest.builder
+
+import com.deck.common.entity.RawNotificationSettingsType
+import com.deck.common.entity.RawTeamType
+import com.deck.common.util.GenericId
+import com.deck.rest.request.*
+
+public class CreateTeamBuilder : RequestBuilder<CreateTeamRequest> {
+    public var name: String? = null
+    public var userId: GenericId? = null
```

Comment:
What does this `userId` mean?

Reply:
The user who created the Team, only accepts the client ID

Reply:
In that case, it shouldn't be on the builder. I have a solution in mind but I need to check it first.